conjur/argparsewrapper.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+
+"""
+Argparsewrapper module
+
+This module is a wrapper for the third-party argparse module used to parse
+commandline arguments. The module controls the types of errors and help screens
+we return according to the types of inputs received.
+"""
+
+import argparse
+import sys
+
+class ArgparseWrapper(argparse.ArgumentParser):
+    """
+    Wrapper to override default behavior of the argparse module.
+    """
+
+    def error(self, message):
+        """
+        This method handles the errors that originate from incorrect
+        commands.
+        """
+        sys.stderr.write('Error %s\n' % message)
+        self.print_help()
+        sys.exit(1)
+
+    @staticmethod
+    def _subcommand_error(message, help_screen):
+        """
+        This method handles the errors that originate from incorrect
+        subcommands/flags.
+        """
+        sys.stderr.write('Error %s\n' % message)
```

Comment:
The linter is telling me otherwise: https://jenkins.conjur.net/blue/organizations/jenkins/cyberark--conjur-api-python3/detail/add_deployment/25/pipeline#step-24-log-1160

what is the diff?